### PR TITLE
Added grpc protocol contents.

### LINKED ISSUE #258: Update docs with better formatting for v2 protocol page
We should do something very similar to https://docs.seldon.io/projects/seldon-core/en/latest/reference/apis/v2-protocol.html ? as far as the formatting goes.

We should replace the hardcoded table in the v2_protocol page https://kserve.github.io/website/master/modelserving/data_plane/v2_protocol/

Thanks @alexagriffith for the suggestions

### PR DESCRIPTION
**Pull request by andyi2it**
"Fixes #258 "

## Proposed Changes 
- Loads the document from kserve url and generates the table structure using the library.

### CODE REVIEW

**Review by alexagriffith — APPROVED** [review]

File: docs/reference/v2_inference/README.md [hunk 1]
```diff
@@ -0,0 +1,373 @@
...
+| Field | Type | Description |
+| ----- | ---- | ----------- |
+| name | [string](#string) | The server name. |
+| version | [string](#string) | The server version. |
+| extensions | [repeated string](#string) | The extensions supported by the server. |
+
+
+
+
+#### ServerReadyRequest
```
Comment (alexagriffith) [inline docs/reference/v2_inference/README.md:355]: @yuzisun this is missing?

Comment (alexagriffith) [reply docs/reference/v2_inference/README.md:355]: @andyi2it

Comment (alexagriffith) [reply docs/reference/v2_inference/README.md:355]: also there is no description under a few of these.. is that intentional?

Comment (alexagriffith) [reply docs/reference/v2_inference/README.md:355]: do we want to add something?

Comment (yuzisun) [reply docs/reference/v2_inference/README.md:355]: This is generated from the proto schema which is empty struct
https://github.com/kserve/open-inference-protocol/blob/main/specification/protocol/open_inference_grpc.proto#L55

**Pull request merged**